### PR TITLE
Some grammar, wording and structural edits combined.

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
Signed-off-by: xiang <xiang13225080@163.com>

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
same with https://github.com/chaos-mesh/chaos-mesh-action/pull/5, PTAL @dcalvin

File: README.md [hunk 1]
```diff
@@ -39,23 +37,23 @@ spec:
...
 
 ```shell
 CFG_BASE64=`base64 chaos.yaml`
 ```
 
-### Create the workflow
+### Step 3. Create the workflow
 
-#### Deploy K8s cluster
+#### Deploy the Kubernetes cluster
```
Comment (dcalvin) [inline README.md]:
```suggestion
1. Deploy the Kubernetes cluster
```

File: README.md [hunk 2]
```diff
@@ -39,23 +37,23 @@ spec:
...
 
-Need to deploy a K8s cluster in the workflow, can use [Kind Cluster](https://github.com/marketplace/actions/kind-cluster) or [Kind Action](https://github.com/marketplace/actions/kind-kubernetes-in-docker-action).
+A Kubernetes cluster is required for the workflow. You can use [Kind Cluster](https://github.com/marketplace/actions/kind-cluster) or [Kind Action](https://github.com/marketplace/actions/kind-kubernetes-in-docker-action) to deploy.
 
 #### Use chaos-mesh-action
```
Comment (dcalvin) [inline README.md]:
```suggestion
2. Use `chaos-mesh-action`.
```

File: README.md [hunk 3]
```diff
@@ -39,23 +37,23 @@ spec:
...
 
-When creating the workflow in Actions, use chaos-mesh/chaos-mesh-action in the yaml configuration file and configure the base64 value of the chaos configuration file. The chaos-mesh related configuration is as follows:
+To create the workflow in GitHub action, use chaos-mesh/chaos-mesh-action in the yaml configuration file and configure the base64 value of the chaos configuration file. The chaos-mesh related configuration is as follows:
```
Comment (dcalvin) [inline README.md]:
```suggestion
To create the workflow in GitHub action, use `chaos-mesh/chaos-mesh-action` in the yaml configuration file and configure the base64 value of the chaos configuration file. The chaos-mesh related configuration is as follows:
```

**Review by dcalvin — APPROVED** [review]
LGTM

**Pull request merged**